src/q2.svg

```diff
@@ -0,0 +1,27 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" preserveAspectRatio="xMidYMid" width="650" height="650" viewBox="0 0 650 650">
```

Comment:
q2 is the best SVG version you have. It's only 1kb. Good job. In general with SVG you want CSS to style the size. So my recommendation here is to remove the width and height attributes.